README.adoc

```diff
@@ -4336,6 +4336,23 @@ Use non-capturing groups when you don't use the captured result.
 /(?:first|second)/
 ----
 
+=== No mix named and numbered captures [[no-mix-named-and-numbered-captures]]
+
+Do not mix named captures and numbered captures in a Regexp literal.
+Because numbered capture is ignored if they're mixed.
+
+[source,ruby]
+----
+# bad
+/(?<foo>FOO)(BAR)/
+
+# good
+/(?<foo>FOO)(?<bar>BAR)/
```

Comment:
It would be better if there is an example of numbered capture.
Ref: https://github.com/rubocop-hq/rubocop/pull/7749/files#r383332351

Reply:
Thanks! I added the example and force pushed.